src/Components/Web/src/WebRenderer.cs

```diff
@@ -18,18 +18,43 @@ namespace Microsoft.AspNetCore.Components.RenderTree
     public abstract class WebRenderer : Renderer
     {
         private readonly IServiceProvider _serviceProvider;
+        private readonly DotNetObjectReference<WebRendererInteropMethods> _interopMethodsReference;
 
         /// <summary>
         /// Constructs an instance of <see cref="WebRenderer"/>.
         /// </summary>
         /// <param name="serviceProvider">The <see cref="IServiceProvider"/> to be used when initializing components.</param>
         /// <param name="loggerFactory">The <see cref="ILoggerFactory"/>.</param>
-        public WebRenderer(IServiceProvider serviceProvider, ILoggerFactory loggerFactory)
+        /// <param name="jsonOptions">The <see cref="JsonSerializerOptions"/>.</param>
+        /// <param name="jsComponentInterop">The <see cref="JSComponentInterop"/>.</param>
+        public WebRenderer(
```

Comment:
You'll have to uncomment https://github.com/dotnet/aspnetcore/blob/main/Directory.Build.props#L18-L23 to get the analyzer to run in VS.